### PR TITLE
Add example for tx with ma operations and update docs for multi assets implementation

### LINKED ISSUE #275: Update docs and examples for MA
# Description

This task involves two main updates:

- The documentation should explain how MA were introduced into Cardano Rosetta
- Create an example for MA transaction generation

## Implementation details

1. The transaction example doesn't need to be complex at all
2. It should follow #274 approach

### PR DESCRIPTION
**Pull request by jvieiro**
# Description

Adds a TypeScript script for sending a transaction with multi assets at `examples/` and also updates the documentation for multi assets implementation.

# References

Closes #275.

### CODE REVIEW

File: examples/commons.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@
 /* eslint-disable camelcase */
 /* eslint-disable new-cap */
 /* eslint-disable no-console */
-import delay from 'delay';
-import * as NaCl from 'tweetnacl';
-import axios from 'axios';
+import delay from "delay";
+import * as NaCl from "tweetnacl";
+import axios from "axios";
```
Comment (AlanVerbner) [inline examples/commons.ts:7]: :thinking:

File: examples/commons.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ const httpClient = axios.create({
 
 const network_identifier = {
   blockchain: "cardano",
-  network: "testnet",
+  network: 3,
```
Comment (AlanVerbner) [inline examples/commons.ts:17]: This shouldn't be changed ... I guess we can test locally and then actually use it when MA arrive to `testnet`

File: docs/rosetta-custom-definitions.md [hunk 3]
```diff
@@ -115,10 +211,68 @@ The following metadata is also returned when querying for block information:
 }
 ```
 
+## `/block/transactions`
```
Comment (AlanVerbner) [inline docs/rosetta-custom-definitions.md:214]: I think we need to add a note that states that transaction assets will be sorted by name

File: examples/commons.ts [hunk 4]
```diff
@@ -123,51 +125,70 @@ const buildOperation = (
...
+      outputAmount = (adaBalance * BigInt(dividend)) / BigInt(100);
       i += 5;
-      if (outputAmount < 2500000) throw new Error(`outputAmount=${outputAmount} is too low. Try with more funds.`);
-    }
-    while (totalBalance - outputAmount <= 2000000)
+      if (outputAmount < 2500000)
+        throw new Error(
+          `outputAmount=${outputAmount} is too low. Try with more funds.`
+        );
+    } while (adaBalance - outputAmount <= 2000000);
```
Comment (AlanVerbner) [inline examples/commons.ts:158]: I don't recall where this numbers came from. I think we should use a constant or something

**Review by AlanVerbner — COMMENTED** [review]
@jvieiro for some reason I'm not seeing as output the address that's supposed to receive the funds, is that ok?

**Comment by jvieiro** [comment]
> @jvieiro for some reason I'm not seeing as output the address that's supposed to receive the funds, is that ok?

Yes, didn't exist a log for the receiver address. But recently I pushed one for that.

**Review by AlanVerbner — APPROVED** [review]

**Comment by AlanVerbner** [comment]
I'm merging as the CI error is not related to actual tests

**Pull request merged**